exercises/sieve/instructions.md

```diff
@@ -0,0 +1,26 @@
+Your task is to create a program that implements the Sieve of Eratosthenes algorithm to find prime numbers.
+
+A prime number is a number that is only divisible by 1 and itself.
+For example, 2, 3, 5, 7, 11, and 13 are prime numbers.
+
+The Sieve of Eratosthenes is an ancient algorithm that works by taking a list of numbers and crossing out all the numbers that aren't prime.
+
+A number that is _not_ prime is called a **composite** number.
```

Comment:
```suggestion
A number that is **not** prime is called a _composite_ number.
```

I'd reverse this because of how rendering works on the website. We want `not` to really stand out and "composite" to be more italic I think. (I'd actually probably choose to put "composite number" in quotes rather than use formatting here.